benchttp/repository.go

```diff
@@ -12,3 +12,14 @@ type ReportService interface {
 	// Retrieve retrieves a Report by ID from the data layer.
 	Retrieve(ctx context.Context, id string) (Report, error)
 }
+
+// ComputedStatsService defines the interface to implement by a
+// service facade inside the application.
+type ComputedStatsService interface {
```

Comment:
```suggestion
type StatsService interface {
```
Stats _are_ computed I think there's no need to prefix.